<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 145, 0);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>REMOVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 145, 0);
border-radius:20px;</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addTask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);
border-radius:20px;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>451</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(119, 118, 123);
border-radius:20px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Remove_all">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 145, 0);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>REMOVE ALL</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>487</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
